modules/mod_whosonline/src/Helper/WhosonlineHelper.php

```diff
@@ -94,7 +94,7 @@ public static function getOnlineUserNames($params)
             ->where($db->quoteName('a.client_id') . ' ' . $whereCondition)
             ->group($db->quoteName(['a.username', 'a.userid', 'a.client_id']));
 
-        $user = Factory::getUser();
+        $user  = Factory::getApplication()->getIdentity();
```

Comment:
```suggestion
        $user = Factory::getApplication()->getIdentity();
```
Remove redundant space.